discord/client.py

```diff
@@ -289,6 +289,7 @@ def __init__(self, *, intents: Intents, **options: Any) -> None:
         self._application: Optional[AppInfo] = None
         self._connection._get_websocket = self._get_websocket
         self._connection._get_client = lambda: self
+        self.__task_store: set[asyncio.Task] = set()
```

Comment:
```suggestion
        self.__task_store: Set[asyncio.Task] = set()
```
Should use old classes for consistency right?

Reply:
Yeah, it should. This has been changed and an import added for it, my fault for forgetting that's not valid in 3.8